uclasm/matching/global_cost_bound/from_local_bounds.py

```diff
@@ -12,4 +12,10 @@ def from_local_bounds(smp):
     """
     costs = smp.local_costs / 2 + smp.fixed_costs
     global_cost_bounds = constrained_lsap_costs(costs)
-    smp.global_costs[:] = global_cost_bounds
+
```

Comment:
Don't do this, do the monotone assignment as it was originally. Otherwise, if we ever have multiple methods that set the global costs, this one will overwrite all the others without doing monotone array comparison.